fenicsxprecice/fenicsxprecice.py

```diff
@@ -7,7 +7,7 @@
 from .config import Config
 import logging
 import precice
-from .adapter_core import FunctionType, determine_function_type, convert_fenicsx_to_precice, get_fenicsx_vertices, CouplingMode, Vertices
+from .adapter_core import FunctionType, determine_function_type, get_fenicsx_vertices, CouplingMode, Vertices, convert_fenicsx_to_precice_coordinateBased
```

Comment:
```suggestion
from .adapter_core import FunctionType, determine_function_type, convert_fenicsx_to_precice_coordinateBased, get_fenicsx_vertices, CouplingMode, Vertices
```
Makes diff easier to read.

Reply:
I will change that